Dockerfile

```diff
@@ -208,6 +208,18 @@ RUN mkdir -p "$RUSTUP_HOME" && chown dependabot:dependabot "$RUSTUP_HOME"
 USER dependabot
 RUN curl https://sh.rustup.rs -sSf | sh -s -- -y \
   && rustup toolchain install 1.51.0 && rustup default 1.51.0
+
+
+### Terraform
+
+USER root
+RUN curl -fsSL https://apt.releases.hashicorp.com/gpg | apt-key add -
+RUN apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main" \
+  && apt-get update -y \
+  && apt-get install terraform \
```

Comment:
Should we pin the version here?

Reply:
Meh, whatevs let's do it in a follow-up PR

Reply:
Pinned to 1.0.0 https://github.com/dependabot/dependabot-core/pull/3766/commits/81a424eac4bd9086a86fe9cd8d18ccf09d5bf2f2